src/test/fixtures/profiles/processed-profile.js

```diff
@@ -853,25 +845,23 @@ export function getNetworkMarkers(options: $Shape<NetworkMarkersOptions> = {}) {
   const startPayload: NetworkPayload = {
     type: 'Network',
     id,
+    startTime,
+    endTime,
     pri: 0,
     status: 'STATUS_START',
-    startTime,
-    endTime: fetchStart,
     URI: uri,
   };
 
   const stopPayload: NetworkPayload = {
     ...startPayload,
     status: 'STATUS_STOP',
-    startTime: fetchStart,
-    endTime,
     contentType: 'text/html',
     ...payload,
   };
 
   return [
-    [name, startTime, startPayload.endTime, startPayload],
-    [name, startTime, stopPayload.endTime, stopPayload],
```

Comment:
This was here before, but shouldn't this `startTime` be `fetchStart` as well?

Reply:
Yes it should!

Apparently I fixed this by the end, here is the final diff:

```
@@ -701,10 +874,8 @@ export function getNetworkMarkers(options: $Shape<NetworkMarkersOptions> = {}) {
   };

   return [
-    // Note that the "time" of network markers is generally close to the
-    // payload's endTime. We don't use it at all in our business logic though.
-    [name, startPayload.endTime, startPayload],
-    [name, stopPayload.endTime, stopPayload],
+    [name, startTime, fetchStart, startPayload],
+    [name, fetchStart, endTime, stopPayload],
   ];
 }
```

I think this is where my struggles were with these marker fixtures.